modules/apps/transfer/keeper/keeper.go

```diff
@@ -7,7 +7,7 @@ import (
 	sdkmath "cosmossdk.io/math"
 
 	"github.com/cosmos/cosmos-sdk/codec"
-	"github.com/cosmos/cosmos-sdk/store/prefix"
+	storeprefix "github.com/cosmos/cosmos-sdk/store/prefix"
```

Comment:
q: linter? We also have this same import path in other places and can't see it being changed (`grep -inr "github.com/cosmos/cosmos-sdk/store/prefix" . --include="*.go"`)